.travis.yml

```diff
@@ -26,7 +26,7 @@ services:
 # ALPAKA_CI                                     : {TRAVIS}
 # ALPAKA_CI_DOCKER_BASE_IMAGE_NAME              : {ubuntu:16.04, ubuntu:18.04}
 # ALPAKA_CI_BOOST_BRANCH                        : {boost-1.65.1, boost-1.66.0, boost-1.67.0, boost-1.68.0, boost-1.69.0, boost-1.70.0, boost-1.71.0}
-# ALPAKA_CI_CMAKE_VER                           : {3.11.4, 3.12.4, 3.13.5, 3.14.7, 3.15.5, 3.16.0}
+# ALPAKA_CI_CMAKE_VER                           : {3.15.7, 3.16.4}
```

Comment:
It is not good that those CMake version changes are now part of this PR. They have already been merged into `develop`. It looks like they have been cherry-picked into this branch which results in new commits with different hashes. There probably have also been some merge conflict resolutions in here.
I am not sure what git/github will do when we merge the branch now. It might skip those changes because they are already applied to develop and keep empty commits.
The correct way would have to rebase this branch onto `develop` (`git fetch && git rebase mainline/develop`). Could you please do this to get rid of those changes here?

Reply:
Please excuse the bugs as well as the merge error, I don't know what happened there. I hope everything is resolved now.